R/connect_to_etn.R

```diff
@@ -1,30 +1,19 @@
 #' Connect to the ETN database
 #'
-#' `r lifecycle::badge("deprecated")`
-#'     Connect to the ETN database using username and password.
+#' This function is `r lifecycle::badge("deprecated")` since etn version 2.3.0.
+#' It's use is no longer supported or needed. All connections to the ETN database
+#' are now made automatically when you use a function. If your credentials are not saved
+#' in the system environement, you will be prompted to enter them.
 #'
-#' @param username Character. Username to use for the connection.
-#' @param password Character. Password to use for the connection.
+#' @return This function is no longer in use, and returns NULL invisibly.
 #'
-#' @return ODBC connection to ETN database.
-#'
-#'
-#' @examples
-#' \dontrun{
-#' # Connect to the ETN database using your rstudio.lifewatch.be username and
-#' # password, and save as the default connection variable "con"
-#' con <- connect_to_etn()
-#'
-#' # Connect to the ETN database using non-default username and password
-#' con <- connect_to_etn(username = "my_username", password = "my_password")
-#' }
-connect_to_etn <- function(username = Sys.getenv("userid"),
-                           password = Sys.getenv("pwd")) {
-  con <- DBI::dbConnect(
-    odbc::odbc(),
-    "ETN",
-    uid = paste("", tolower(username), "", sep = ""),
-    pwd = paste("", password, "", sep = "")
+#' @export
+connect_to_etn <- function() {
```

Comment:
I notice the params have been removed as well. How will this react when a user calls:

```R
connect_to_etn(username = "foo", password = "bar")
# or
connect_to_etn("foo", "bar")
```

Reply:
`Error in connect_to_etn("foo", "bar") : unused arguments ("foo", "bar")`

We can get around this with ellipsis, well spotted.